src/wazuh_modules/vulnerability_scanner/qa/test_policy_change_log.py

```diff
@@ -0,0 +1,165 @@
+            file.unlink()
+        Path("queue/indexer/wazuh-states-vulnerabilities").rmdir()
+
+
+    # Set the path to the binary
+    cmd = Path("build/wazuh_modules/vulnerability_scanner/testtool/scanner/", "vd_scanner_testtool")
+    cmdAlt = Path("wazuh_modules/vulnerability_scanner/build/testtool/scanner/", "vd_scanner_testtool")
+
+    # Ensure the binary exists
+    if not cmd.exists():
+        cmd = cmdAlt
+    assert cmd.exists(), "The binary does not exists"
+
+    # Remove previous log file if exists
+    if Path("log.out").exists():
+        Path("log.out").unlink()
+
+    test_folder = request.param
+    LOGGER.debug(f"Running test {test_folder}")
+
+    json_files = sorted(Path(test_folder).glob("args_*.json"))
+    for json_file in json_files:
+        LOGGER.debug(f"Running test {json_file}")
+
+        # Read arguments from the json file
+        with open(json_file) as f:
+            args = json.load(f)
+
+        command = [cmd] + args
+
+        process = subprocess.Popen(command)
+
+        # Check if the process is initialized finding in the log file the line "Vulnerability scanner module started"
+        start_time = time.time()
+        log_file = "log.out"
+
+        while not Path(log_file).exists() and (time.time() - start_time <= 10):
+            time.sleep(1)
+
+        # Check if the log file exists, if the line is not found, try again in 1 second
+        assert Path(log_file).exists(), "The log file does not exists"
+
+        LOGGER.debug(f"Wating for the process to be initialized")
+        found = find_regex_in_file(r"Vulnerability scanner module (started|is disabled)", log_file)
+        assert found, "The process is not initialized"
+        LOGGER.debug(f"Process initialized")
+
+        # Regex to extract the value of the *** in args_***.json
+        regex = r"args_(\d+).json"
+        match = re.search(regex, json_file.name)
+        assert match, "The regex does not match"
+        expected_json_file = Path(test_folder, f"expected_{match.group(1)}.out")
+        assert expected_json_file.exists(), "The expected file does not exists"
+
+        expected_lines = []
+
+        json_data = json.load(open(expected_json_file))
+        LOGGER.debug(f"Expected json data: {json_data}")
+        for line in json_data:
+            expected_lines.append(line)
+
+        LOGGER.debug(f"Expected lines: {expected_lines}")
+        quantity_expected_lines = len(expected_lines)
+        LOGGER.debug(f"Quantity expected lines: {quantity_expected_lines}")
+
+        found_lines = {line: False for line in expected_lines}
+        timeout = 10
+
+        basetimeout = timeout
+        for expected_line in expected_lines:
+            while not found_lines[expected_line]:
+                if timeout < 4*basetimeout:
+                    tail_log(log_file, expected_lines, found_lines, timeout)
+                    timeout = 1.5*timeout
+
+        LOGGER.debug(f"Succesfully found all expected lines")
```

Comment:
Unnecessary format and typo
```suggestion
        LOGGER.debug("Successfully found all expected lines")
```